### PR TITLE
feat(extensions): add 1loc

### PR DESCRIPTION
**Pull request by kud**
## Description

Browse https://1loc.dev/

## Type of change

- New extension

## Screencast

https://cln.sh/Dqva1V9J7SJXGLskWGpe

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/1loc/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "name": "1loc",
+  "title": "1loc",
+  "description": "Browsing 1loc.dev",
+  "icon": "command-icon.png",
+  "author": "kud",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "1loc",
```
Comment (pernielsentikaer) [inline extensions/1loc/package.json:12]:
```suggestion
      "title": "Search Javascript Utilities",
      "subtitle": "1LOC",
```
I'm not sure about the title, but something like that should be changed 😄

File: extensions/1loc/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "description": "Browsing 1loc.dev",
```
Comment (pernielsentikaer) [inline extensions/1loc/package.json:13]:
```suggestion
      "description": "Browse Favorite JavaScript Utilities from 1LOC - single line of code! No more!",
```

File: extensions/1loc/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "1loc",
+  "title": "1loc",
+  "description": "Browsing 1loc.dev",
```
Comment (pernielsentikaer) [inline extensions/1loc/package.json:5]:
```suggestion
  "description": "Browsing 1LOC - Favorite JavaScript Utilities in single line of code! No more!",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @kud 💣 

Thanks for an awesome extension, looking forward to use it myself 💯 

Your extensions suffers of [Empty state flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states) - could you address this?

I'll do another review when this is corrected 💪 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by kud** [comment]
Fixed @pernielsentikaer

File: extensions/1loc/package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "1loc",
+  "title": "1loc - JavaScript Utilities in single line of code",
```
Comment (pernielsentikaer) [inline extensions/1loc/package.json:4]:
```suggestion
  "title": "1 LOC - JavaScript Utilities in single line of code",
```

File: extensions/1loc/package.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  "description": "Browse JavaScript Utilities from 1loc.dev",
+  "icon": "command-icon.png",
+  "author": "kud",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "1loc - JavaScript Utilities in single line of code",
```
Comment (pernielsentikaer) [inline extensions/1loc/package.json:12]:
```suggestion
      "title": "Search JavaScript Utilities",
      "subtitle": "1 LOC"
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @kud 👋 

I think you changed too much 😄 

Could you look into something like the suggested?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by kud** [comment]
Thank you @sxn  :)

**Pull request merged**